src/gluonts/model/n_beats/_ensemble.py

```diff
@@ -363,9 +364,25 @@ def __init__(
         )
         self.meta_bagging_size = meta_bagging_size
 
+        # Handle 'sMAPE' specific learning issues though learning rate adjustment
+        assert self.meta_loss_function is not None and (
+            isinstance(self.meta_loss_function, list)
+            or isinstance(self.meta_loss_function, tuple)
+        )
+        if trainer is None:
+            if "sMAPE" in self.meta_loss_function:
+                trainer = Trainer(learning_rate=1e-4)
+            else:
+                trainer = Trainer()
+        assert trainer is not None
+        if trainer.learning_rate > 1e-4 and "sMAPE" in self.meta_loss_function:
+            logging.warning(
+                "You may experience issues when using 'sMAPE' as "
+                "loss-function and a learning rate above 1e-4."
+            )
```

Comment:
I'm wondering if we want to keep this warning here. The thing is, with any loss you'll get problems if the learning rate is too large. Maybe having the ad-hoc default learning rate for "sMAPE" is fine for now.

Same thing applies to the _estimator.py below

Reply:
Thats true, however, in this case it actually makes all the difference. So the second evaluation with a learning rate of 1e-4 is done, and now all the weird behavior is gone, so thats like a night and day difference. 

So if I was doing my experience as a customer and I wanted to modify some property of the trainer (but not changing the learning rate, aka keeping the default of 1e-3), I would suddenly get super weird behavior, I would very much appreciate to know why and not have to debug by myself. The warning would in this case save a lot of headache.

In fact, looking at the experiments, I think we could make the default learning rate for all loss functions 1e-4.